cdapython/fetch.py

```diff
@@ -1319,7 +1307,7 @@ def fetch_rows(
     # If we're adding extra columns from some non-`table` table*, always
     # include that table's ID field, whether or not it was requested.
     # 
-    # *...unless it's `somatic_mutation`, from which we can add extra columns
+    # *...unless it's `mutation`, from which we can add extra columns
     # but which doesn't have its own ID field.
```

Comment:
this comment no longer applies; `mutation` has its own ID field now and `join_table_id_field` gets set properly by the new code near (old line number) 741